platform/reference/migrations/4-0-migration.mdx

```diff
@@ -0,0 +1,129 @@
+the vCluster would automatically upgrade these virtual clusters to the latest default vCluster version. Due to the [breaking changes introduced in v0.20](https://roadmap.vcluster.com/changelog/vcluster-v020-ga), 
+automatic upgrades of vCluster version are no longer supported and a vCluster version must be set in the spec of a virtual cluster or virtual cluster template. 
+
+We recommend confirmging that the versions of your virtual clusters and virtual cluster templates are in the resource yaml before upgrading. Otherwise,
+your virtual clusters will have errors requiring you to define these versions post upgrade and since it will not know what version you upgraded from, it will not know what
+version to set and manual steps will need to happen to determine the correct version and set it. 
+
+#### How to define the vCluster version
+
+In the UI, type the version of the virtual cluster or virtual cluster template in the textbox and the resource yaml will automatically update the spec to include the version.
+
+If the resource yaml of your virtual cluster or virtual cluster template has `spec.template.helmRelease.chart.version` set, then the virtual cluster version is defined.
+
+`spec` Example:
+
+```
+spec:
+  template:
+    helmRelease:
+      chart:
+        version: 0.19.7
+```
+
+## Download the latest vCluster CLI (v0.20.0+)
+
+With the deprecation of the Loft CLI, review the [Loft CLI to vCluster CLI Migration docs](https://vcluster.com/docs/vcluster/reference/migrations/loft-cli-vcluster-cli-migration).
+
+## Required Changes in your Platform Configuration before Upgrading
+
+A new feature has been added to support the ability to change the prefix of the namespace created for projects. In previous versions, the 
+prefix was `loft-p`, but in v4.0.0, the new default is `p-`. Due to the change in defaults, the previous prefix needs to be set in the Platform configuration as 
+part of the upgrade command. Since future upgrades reuse the values of the configuration, you only need to apply it once.
+
+### Appending the project prefix to upgrade commands
+
+<Tabs
+  defaultValue="cli"
+  values={[
+    { label: 'CLI', value: 'cli', },
+    { label: 'helm', value: 'helm', },
+  ]
+}>
+<TabItem value="cli">
+
+To upgrade the Platform from v3 to v4 via vCluster CLI, run. Update `$PLATFORM_VERSION` with a valid vCluster Platform version.
+
+:::info Plaftorm Namsespace
+The following steps assume the existing installation is in the default namespace `loft`. The default namespace for v4 is vcluster-platform, so we specify the namespace in the steps.
+:::
+
+```bash
+PLATFORM_NAMESPACE="loft" # update if the platform is installed to a different namespace
```

Comment:
These should be `export PLATFORM_NAMESPACE=loft` and `export PLATFORM_VERSION=<PLATFORM_VERSION>` instead

Reply:
@sowmyav27 As discussed offline, I don't believe we need to change this and it works as expected, so I'm going to resolve it.